scalafmt-cli/src/main/scala/org/scalafmt/cli/CliArgParser.scala

```diff
@@ -161,4 +175,12 @@ object CliArgParser {
   def buildInfo =
     s"""build commit: ${Versions.commit}
        |build time: ${new Date(Versions.timestamp.toLong)}""".stripMargin
+
+  private implicit val urlOptRead: Read[URL] = Read.reads {
+    Try(new URL(_)).fold(
+      _ => throw new IllegalArgumentException("Expected a URL"),
+      identity
+    )
```

Comment:
I am about to commit this implicit to scopt

Reply:
why swallow the `URL` exception? I think it would be clearer to report the original error message.